<?xml version="1.0"?>

<launch>
    <node name="arduino" pkg="rosserial_python" type="serial_node.py" output="screen">
        <param name="port" value="/dev/ttyACM0" />
    </node>

    <node name="pipe_reader" pkg="joy_convert" type="pipe_reader.py" output="screen" >
        <param name="pipe_name" value="/tmp/xy" />
        <param name="hz" value="20" />
        <remap from="x" to="/image_x" />
        <remap from="y" to="/image_y" />
    </node>

    <node name="image_transform_x" pkg="joy_convert" type="image_transform.py" output="screen" >
        <param name="minAngle" value="1250" />
        <param name="maxAngle" value="1800" />
        <param name="minPixel" value="0" />
        <param name="maxPixel" value="1280" /> 

        <remap from="input" to="/image_x" />
        <remap from="servo" to="/hip_servo" />
    </node>

    <node name="image_transform_y" pkg="joy_convert" type="image_transform.py" output="screen" >
        <param name="minAngle" value="1250" />
        <param name="maxAngle" value="1800" />
        <param name="minPixel" value="0" />
        <param name="maxPixel" value="720" /> 

        <remap from="input" to="/image_y" />
        <remap from="servo" to="/shoulder_servo" />
    </node>


</launch>
